client/js/components/statement/splitStatement/FloatingContextButton.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <button
+    v-show="isVisible"
+    class="bg-white rounded shadow absolute p-0.5"
+    :aria-controls="section"
+    :aria-expanded="isContentCollapsed"
+    :data-cy="`sidebar:floatingContextButton:${section}`"
+    @click="toggleContentVisibility"
+    @mouseover="show"
+    @mouseleave="hide">
+    <dp-icon
+      :aria-label="Translator.trans('content.show/hide')"
+      class="w-4 h-4 rounded-sm text-blue-700 hover:bg-gray-100"
```

Comment:
@spiess-demos is this one of the colors, wich will not work anymore, after https://github.com/demos-europe/demosplan-ui/pull/932 finds its way into core? Then we should use something else from the beginning on.

Reply:
yes, both `text-blue-700` and `bg-gray-100` are tailwind defaults. If we'd manage to release a new demosplan-ui version, and bump demosplan-core with it, we could use the correct colors here.